north-star-vision.md

```diff
@@ -0,0 +1,139 @@
+# North Star Vision
```

Comment:
I had to look up what "North Star" meant (and I, uh, _work_ at Microsoft, at least technically) and I am sure I'm not the only one.

It appears to originate as a company jargon term: https://devblogs.microsoft.com/oldnewthing/20151103-00/?p=91861

For more global accessibility, can we just call it "Roadmap Proposal"?

Reply:
Yeah I think we can just call it Roadmap and/or Vision. WDYT, @LappleApple @justaugustus @hasheddan @jeremyrickard?

Reply:
I agree with @guineveresaenger.

Roadmap (Vision) sounds like a great name for this document.

Reply:
Renamed the document to "Roadmap and Vision"